config/initializers/rack_attack.rb

```diff
@@ -24,7 +24,7 @@ class Rack::Attack
   #
   # Key: "rack::attack:#{Time.now.to_i/:period}:req/ip:#{req.ip}"
   throttle("req/ip", limit: 1500, period: 5.minutes) do |req|
-    req.ip # unless req.path.start_with?('/assets')
+    req.env['HTTP_CF_CONNECTING_IP'] || req.ip # unless req.path.start_with?('/assets')
```

Comment:
I think it's better to use `req.remote_ip`, refer:

https://github.com/rails/rails/blob/main/actionpack/lib/action_dispatch/middleware/remote_ip.rb#L123
https://developers.cloudflare.com/fundamentals/get-started/reference/http-request-headers/#x-forwarded-for

Reply:
Here `req` is `Rack::Request` object, not the one in actionpack.
`Rack::Request#ip` has already handled `HTTP_X_FORWARD_FOR`.